### DISCUSSION TITLE
Toggle features

### DISCUSSION THREAD

**Discussion by Gustafssonz** [rxhanson/Rectangle#1003] [Ideas]
When pressing the buttons again, it could lead to:
(Just like Spectacles)
Toggles between 3 different sizes, small -> normal -> large (and back to small) but always default on Normal.

**Comment by rxhanson** [comment]
You can select this behavior in the settings tab of the preferences window at "Repeated commands".

**Reply by Gustafssonz** [reply]
Thank you! I missed that 😄
